### ISSUE TITLE
More prominent Firefox branding in the Nav

### ISSUE THREAD

**Issue by pwalm** [mozilla/addons-frontend#3763]
**Very high priority**: We need to change the puzzle piece to the Firefox icon and adjust our typography to match the Firefox branding guidelines. This needs to be done and launched before the 57 launch.

A good example from Test Pilot:
<img width="1044" alt="screen shot 2017-11-02 at 12 56 38 pm" src="https://user-images.githubusercontent.com/6797635/32347259-4a950a4a-bfcd-11e7-983c-7957dd3daf65.png">

Here is a mock up:
<img width="1253" alt="screen shot 2017-11-02 at 12 56 14 pm" src="https://user-images.githubusercontent.com/6797635/32347271-536b73b6-bfcd-11e7-8f9a-3d4d930007fb.png">

Assets and guidelines [can be found on the Design System](http://design.firefox.com/photon/visuals/product-identity-assets.html).

Spec:
<img width="961" alt="screen shot 2017-11-02 at 1 00 56 pm" src="https://user-images.githubusercontent.com/6797635/32347446-e898e644-bfcd-11e7-9da9-23b4c107eaab.png">

**Comment by kumar303** [comment]
@pwalm I have a few questions:

- The mock you posted has a bit of a pink tint to it that I don't see in the assets. Should I use the official svg asset instead?
- The 36px left margin you spec'd differs from our current 24px margin. Do you really want to change that to 36px? If so, it will look odd if we don't change all margins to 36px, I think.
- Could you give some context as to why we can't use the puzzle icon? I thought it added a unique personality to the site.

**Comment by pwalm** [comment]
- Use the official svg icon [from here](http://design.firefox.com/photon/visuals/product-identity-assets.html).
- Stick with 24px! Sorry, thought we were using 36px. 
- I thought so too, but they want to keep Firefox sites uniform and branded with the Firefox icon. We can still keep the favicon though. ;)

**Comment by muffinresearch** [comment]
Cross-posting this from the PR to keep track of the changes. The following is combined wordmark svg based on @pwalm's spec - this solves one of the issues in that we can use this across all locales and just localize the alternate text to keep visual consistency. (See PR for the additional comments).

Example screenshot in InkScape:
<img width="1124" alt="firefox-addons-logo-horizontal-lockup-1_svg_-_inkscape" src="https://user-images.githubusercontent.com/1514/32387241-bc8c966a-c0bb-11e7-8652-d2c6f7b2468a.png">

Svg in zip file:
[firefox-addons-min.svg.zip](https://github.com/mozilla/addons-frontend/files/1441909/firefox-addons-min.svg.zip)

**Comment by AlexandraMoga** [comment]
Verified as fixed on AMO-dev with FF56 (Win 10 and Android 7.0)

![image](https://user-images.githubusercontent.com/31961530/32448624-170fd300-c318-11e7-8909-b94ca81b3e85.png)

![image](https://user-images.githubusercontent.com/31961530/32449006-1f7c58c8-c319-11e7-8deb-690a32a037c0.png)

**Comment by pwalm** [comment]
Will this be live for the Quantum launch next week? 

Let me rephrase: this needs to be live for the Quantum launch next week 😁

**Comment by willdurand** [comment]
> Will this be live for the Quantum launch next week?

It is live on Thursday 😉

**Comment by pwalm** [comment]
Woo! Thanks all 😁
